src/lib/core/TLVReader.h

```diff
@@ -857,6 +857,13 @@ class DLL_EXPORT TLVReader
      */
     TLVBackingStore * GetBackingStore() { return mBackingStore; }
 
+    /**
+     * Returns true if the current TLV element type is a double.
+     *
+     * @return @p true if the current TLV element type is a double; otherwise @p false.
```

Comment:
`@return` line is redundant. The doc comment already describes what it does fully (and the name of `IsElementDouble` is quite obvious as well)

Reply:
removed